api/envoy/config/filter/network/find_and_replace/v2alpha1/find_and_replace.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+package envoy.config.filter.network.find_and_replace.v2alpha1;
+option go_package = "v2alpha1";
+
+import "validate/validate.proto";
+import "gogoproto/gogo.proto";
+
+// [#protodoc-title: Find and Replace]
+// Find and Replace Network Filter :ref:`configuration overview
+// <config_network_filters_find_and_replace>`.
+
+// Find and Replace network filter config.
+//
+message FindAndReplace {
```

Comment:
Can you add more complete docs here so that it's easier to understand at the config layer how this filter would be used? (Obviously in the final PR we will need complete docs, but let's start here w/ the protos). I suspect there is going to be overlap with tap/dump matching, and I want to make sure we future proof the matching part of this so that we can use it in both places.

Reply:
Done (at least for this initial config).